### PR TITLE
ContextualMenu: Updated styles for expanded menu item to match toolkit

### LINKED ISSUE #3242: Contextual menu selected items are not to spec
Are you willing to submit a PR to fix? (Yes)

Requested priority: (Normal) 

Products/sites affected: (if applicable)
OneDrive and SharePoint. Soon to be Outlook

The selected items currently have a grey background and blue check. There should be no background and neutral check.
Build:
image

Spec:
image

### PR DESCRIPTION
**Pull request by lynamemi**
#### Pull request checklist

- [X] Addresses an existing issue: Fixes #3242
- [X] Include a change request file using `$ npm run change`

#### Description of changes

- Updated styles for expanded menu item to match toolkit.
- The semantic slot update will only effect ContextualMenu. A [PR](https://github.com/OfficeDev/office-ui-fabric-react/pull/3377) I have out for ComboBox removes its reference to that slot.
- Font weight does not change in the expanded state.
- ContextualMenuCustomizationExample was overwritten in a recent PR. Putting it back.

Before
contextualmenu before

After
image

#### Focus areas to test

(optional)

### CODE REVIEW

File: common/changes/office-ui-fabric-react/contextualmenu-bug_2017-11-21-19-43.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "ContextualMenu: Updated exampanded menu item style to match toolkit.",
```
Comment (phkuo) [inline common/changes/office-ui-fabric-react/contextualmenu-bug_2017-11-21-19-43.json]: "expampanded"
spelling

File: packages/office-ui-fabric-react/src/common/_semanticSlots.scss [hunk 2]
```diff
@@ -40,7 +40,7 @@ $buttonTextCheckedHoveredColor: "[theme:buttonTextCheckedHovered, default: #0000
 
 /* Menus */
 $menuItemBackgroundHoveredColor: "[theme:menuItemBackgroundHovered, default: #f8f8f8]";
-$menuItemBackgroundCheckedColor: "[theme:menuItemBackgroundChecked, default: #d0d0d0]";
+$menuItemBackgroundCheckedColor: "[theme:menuItemBackgroundChecked, default: #eaeaea]";
```
Comment (phkuo) [inline packages/office-ui-fabric-react/src/common/_semanticSlots.scss]: i thought we were going to set it to no background?

Comment (lynamemi) [reply packages/office-ui-fabric-react/src/common/_semanticSlots.scss]: Can you elaborate?

File: common/changes/office-ui-fabric-react/contextualmenu-bug_2017-11-21-19-43.json (hunk 1 above)
Comment (lynamemi) [reply common/changes/office-ui-fabric-react/contextualmenu-bug_2017-11-21-19-43.json]: example + expanded! 😄

File: packages/office-ui-fabric-react/src/common/_semanticSlots.scss (hunk 2 above)
Comment (phkuo) [reply packages/office-ui-fabric-react/src/common/_semanticSlots.scss]: @lynamemi I thought the design was to have it have a white, blank background, but ignore me if that's the case.

Comment (phkuo) [reply packages/office-ui-fabric-react/src/common/_semanticSlots.scss]: nevermind, I thought this was something else

**Pull request merged**